### ISSUE TITLE
some questions about Few-NERD(intra) setting

### ISSUE THREAD

**Issue by zyccyz** [thunlp/Few-NERD#10]
Hi, I use the script in inter setting to run the Few-NERD(intra) task, but the best result of 5 way 1~2 shot is just 18.47 F1(35.92 reported in the paper). I think it might have something to do with the choice of the hyperparameters. Can you release the script about the intra task?

**Comment by cyl628** [comment]
hi, you can try `--lr 1e-4`. We found a smaller learning rate will have significant impact on nnshot and structshot model. Increasing the batch size to 8 might also help. Thanks for the inquiry, I will update the script with the newest hyperparameters.

**Comment by zyccyz** [comment]
Thank you! This is really helpful!
